### PR TITLE
fix(ui5-input): remove field value when cleared in change event handler

### PR DESCRIPTION
**Pull request by MapTo0**
FIXES: #6249

### CODE REVIEW

**Review by elenastoyanovaa — CHANGES_REQUESTED** [review]
Could you please add a test?

File: packages/main/src/Input.ts [hunk 1]
```diff
@@ -780,6 +783,19 @@ class Input extends UI5Element implements SuggestionComponent, IFormElement {
 		} else {
 			this.closePopover();
 		}
+
+		if (this._performTextSelection) {
+			// modify DOM directly due to some issues with lit rendering
```
Comment (elenastoyanovaa) [inline packages/main/src/Input.ts:788]: Could you please write something more specific, as this comment is not that useful?

File: packages/main/test/pages/Input.html [hunk 2]
```diff
@@ -332,6 +332,11 @@ <h3>Input in a Dialog</h3>
 
 	<h3>Short Input With Centered Text</h3>
 	<ui5-input class="short" value="244"></ui5-input>
+	<ui5-input id="change-event-value" show-suggestions>
```
Comment (elenastoyanovaa) [inline packages/main/test/pages/Input.html:335]: This input field is just pasted in the middle of the test page next to some input. Please be consistent and add it somewhere separately and add a label or a title on top of it to indicate what could be tested here.

File: packages/main/test/pages/MultiInput.html [hunk 3]
```diff
@@ -258,7 +258,7 @@ <h1>Tokens + Suggestions</h1>
 	</div>
 
 	<div class="sample-container">
-		<ui5-multi-input show-suggestions id="token-unique" style="width: 50%">
+		<ui5-multi-input show-suggestions id="token-unique" show-value-help-icon style="width: 50%">
```
Comment (elenastoyanovaa) [inline packages/main/test/pages/MultiInput.html:261]: the same comment as in the input test page.

**Review by elenastoyanovaa — CHANGES_REQUESTED** [review]
Please add a comment in the PR and give more context.

File: packages/main/test/pages/MultiInput.html (hunk 3 above)
Comment (MapTo0) [reply packages/main/test/pages/MultiInput.html:261]: I am just extending the sample

**Review by elenastoyanovaa — APPROVED** [review]

**Pull request merged**